cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -27,8 +26,20 @@ <h2 class="header-slug">
         </span>
     </h2>
     {% endif %}
-    {#% TODO: Add condition to display various related post types %#}
-    {{ _related_posts_list() }}
+    {% set related_post_types = page.related_posts %}
+    {% if related_post_types | length  > 1 %}
```

Comment:
I'm not understanding the point of this logic. If there's more than one type that's selected to relate to the current page, then render the title of each of them, but if there's only one then don't? Why wouldn't we want to show the type of data that's there?